config/rubocop.yml

```diff
@@ -19,15 +19,14 @@ Style/StringLiterals:
 Metrics/LineLength:
   Max: 100
```

Comment:
Another strategy that works well with rubocop metrics is to set the limit to exactly what your ideal threshold would be (in my case I would pick 79 chars), and then add exceptions for anything that does not meet that. It means having more Exclude entries, but it means that all new code will immediately be flagged so you can deal with it right away. Basing thresholds off legacy code usually doesn't provide as nice of a feedback loop as you will often be fixing recent code when you begin to clean things up -- better to draw the line in the sand now and proceed with the higher standards than trying to fix things later.